.gitignore

```diff
@@ -15,3 +15,4 @@
 /libpeerconnection.log
 npm-debug.log
 testem.log
+/yarn.lock
```

Comment:
and then we can check this file in so people that developing on the addon will get the same deps